ercc/go.mod

```diff
@@ -1,29 +1,28 @@
 module github.com/hyperledger-labs/fabric-private-chaincode/ercc
 
-go 1.13
+go 1.14
```

Comment:
Could you do a `git mod tidy` before checking in updated go.mods?  Right now all my go.mod/go.sum are updated, even if i do only a `make clean`!  (This is true also for the top-level go.mod ...)

Reply:
I actually did that! When I run `go mod tidy` right now it does not change anything?! Not sure what to do here?!

Reply:
should I just revert all changes to `go.mod`?

Reply:
well, i had this issue already beforehand. Thought tidy might solve it but apparently not. Following is the diff i get after a build
```
msteiner@ubusrv1804-dev ~/.../src/github.com/hyperledger-labs/fabric-private-chaincode (pr/322 *)$ git diff
diff --git a/go.mod b/go.mod
index ab422b3..9add882 100644
--- a/go.mod
+++ b/go.mod
@@ -23,7 +23,7 @@ require (
        golang.org/x/net v0.0.0-20200320220750-118fecf932d8 // indirect
        golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
        golang.org/x/sys v0.0.0-20200320181252-af34d8274f85 // indirect
-       golang.org/x/tools v0.0.0-20200323164354-18ea2c8f7359 // indirect
+       golang.org/x/tools v0.0.0-20200408132156-9ee5ef7a2c0d // indirect
        google.golang.org/genproto v0.0.0-20200319113533-08878b785e9c // indirect
        google.golang.org/grpc v1.28.0 // indirect
        gopkg.in/ini.v1 v1.55.0 // indirect
diff --git a/go.sum b/go.sum
index 6a0dd40..470e2fc 100644
--- a/go.sum
+++ b/go.sum
@@ -323,6 +323,7 @@ github.com/xeipuuv/gojsonschema v0.0.0-20180618132009-1d523034197f/go.mod h1:5yf
 github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2/go.mod h1:UETIi67q53MR2AWcXfiuqkDkRtnGDLqkBTpCHuJHxtU=
 github.com/xordataexchange/crypt v0.0.3-0.20170626215501-b2862e3d0a77/go.mod h1:aYKd//L2LvnjZzWKhF00oedf4jCCReLcmhLdhm1A27Q=
 github.com/yuin/goldmark v1.1.25/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
+github.com/yuin/goldmark v1.1.27/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
 go.etcd.io/bbolt v1.3.2/go.mod h1:IbVyRI1SCnLcuJnV2u8VeU0CEYM7e686BmAb1XKL+uU=
 go.opencensus.io v0.22.0/go.mod h1:+kGneAE2xo2IficOXnaByMWTGM9T73dGwxeWcUqIpI8=
 go.uber.org/atomic v1.4.0 h1:cxzIVoETapQEqDhQu3QfnvXAV4AlzcvUCxkVUFw3+EU=
@@ -426,6 +427,8 @@ golang.org/x/tools v0.0.0-20191029190741-b9c20aec41a5/go.mod h1:b+2E5dAYhXwXZwtn
 golang.org/x/tools v0.0.0-20191119224855-298f0cb1881e/go.mod h1:b+2E5dAYhXwXZwtnZ6UAqBI28+e2cm9otk0dWdXHAEo=
 golang.org/x/tools v0.0.0-20200323164354-18ea2c8f7359 h1:7R5qvs7puR7tYevClYCMHTOFM3jJbRl2dbKOfmNw7dE=
 golang.org/x/tools v0.0.0-20200323164354-18ea2c8f7359/go.mod h1:Sl4aGygMT6LrqrWclx+PTx3U+LnKx/seiNR+3G19Ar8=
+golang.org/x/tools v0.0.0-20200408132156-9ee5ef7a2c0d h1:2DXIdtvIYvvWOcAOsX81FwOUBoQoMZhosWn7KjXEl94=
+golang.org/x/tools v0.0.0-20200408132156-9ee5ef7a2c0d/go.mod h1:EkVYQZoAsY45+roYkvgYkIh4xh/qjgUK9TdY2XT94GE=
 golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7/go.mod h1:I/5z698sn9Ka8TeJc9MKroUUfqBBauWjQqLJ2OPfmY0=
 golang.org/x/xerrors v0.0.0-20191011141410-1b5146add898/go.mod h1:I/5z698sn9Ka8TeJc9MKroUUfqBBauWjQqLJ2OPfmY0=
 golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 h1:E7g+9GITq07hpfrRu66IVDexMakfv52eLZ2CXBWiKr4=
```

Reply:
Note, though, this time i don't seem to get it anymore for ercc but only in the top-level. On top-level i have it now since quite a while. I got it also at some point-in-time for ercc but not anymore now. So probably your change is fine. But as you update